docs/changelog.rst

```diff
@@ -87,6 +89,8 @@ Changes:
   :bug:`3084`
 
 .. _Groups: https://groups.google.com/forum/#!searchin/beets-users/mbsync|sort:date/beets-users/iwCF6bNdh9A/i1xl4Gx8BQAJ
+=======
+  :bug:`2442`
```

Comment:
Looks like this line might have gotten into the wrong place?

Reply:
Oops, must've missed it during the rebase, fixed!